snmp/tests/test_profiles.py

```diff
@@ -48,6 +48,16 @@
 pytestmark = pytest.mark.usefixtures("dd_environment")
 
 
+def test_load_profiles(aggregator):
+    instance = common.generate_instance_config([])
+    check = SnmpCheck('snmp', {}, [instance])
+    for name, profile in check.profiles.items():
+        try:
+            check._config.refresh_with_profile(profile)
+        except ConfigurationError as e:
+            pytest.fail("Profile `{}` is not configured correctly: {}".format(name, e))
```

Comment:
would it make sense to use this in the `run_profile_check()` too? 

The use case I'm thinking about is when I'm adding a test for a new profile, I usually run a specific test I'm working on which would error out if the profile is wrong. The test_profiles has many profile tests and I might not remember to run `test_load_profiles()` to verify the profile first.